apigee_sync.go

```diff
@@ -517,3 +529,18 @@ func mapIsSubset(a map[string]bool, b map[string]bool) bool {
 
 	return true
 }
+
+/*
+ * Returns true if the two arrays have matching contents
+ */
+func ArrayEquals(a, b []string) bool {
```

Comment:
I saw that. We reflect.deepequal() is not very efficient. For simple checks, we should avoid it. http://www.tapirgames.com/blog/golang-slice-comparison